src/ifp/README.md

```diff
@@ -1,27 +1,43 @@
 # Initialize Floorplan
 
+This tool initializes floorplan constraints, die/core area, and makes tracks. 
+
 ## Commands
 
+```{note}
+- Parameters in square brackets `[-param param]` are optional.
+- Parameters without square brackets `-param2 param2` are required.
+```
+
 ### Initialize Floorplan
 
-```
+```tcl
 initialize_floorplan
-  [-sites site_names]             list of sites to make rows for
-  -die_area "lx ly ux uy"         die area in microns
-  [-core_area "lx ly ux uy"]      core area in microns
-or
-  -utilization util               utilization (0-100 percent)
-  [-aspect_ratio ratio]           height / width, default 1.0
-  [-core_space space
-    or "bottom top left right"]   space around core, default 0.0 (microns).
-                                  Should be either one value for all margins
-                                  or 4 values, one for each margin.
+  [-utilization util]
+  [-aspect_ratio ratio]
+  [-core_space space | {bottom top left right}]
+  [-die_area {lx ly ux uy}]
```

Comment:
@luarss 
`-die_area`/`-core_area` used for fixed value from user.
`-utilization`/`-aspect_ratio` used for automatic die size calculation based on synthesis results. 
So split both with `or` word.

Reply:
Do you recommend we change the function signature or just indicate in docs?

Reply:
I don't understand your question. But you may refer following example and update it to use either one at a time for `initialize_floorplan`
with die/core area: https://github.com/The-OpenROAD-Project/OpenROAD/blob/master/src/ifp/test/init_floorplan1.tcl
with utilization: https://github.com/The-OpenROAD-Project/OpenROAD/blob/master/src/ifp/test/init_floorplan2.tcl#L7

Reply:
I mean like do we want to force a either-or argument like this 
initialize_floorplan
    [-die_area die_area | -core_area core_area]

Reply:
Maybe following format:
```
initialize_floorplan
  [-utilization util]
  [-aspect_ratio ratio]
  [-core_space space | {bottom top left right}]
   [-sites site_name]
or
initialize_floorplan
  [-die_area {lx ly ux uy}]
  [-core_area {lx ly ux uy}]
  [-sites site_name]
```

Reply:
@vijayank88 Updated